crates/aptos/src/move_tool/fmt.rs

```diff
@@ -0,0 +1,147 @@
+    /// priority over .movefmt.toml.
+    /// Config options can be found at https://github.com/movebit/movefmt/blob/develop/doc/how_to_use.md
+    #[clap(long, value_parser = crate::common::utils::parse_map::<String, String>, default_value = "")]
+    pub(crate) config: BTreeMap<String, String>,
+
+    #[clap(long, short)]
+    /// Print verbose output
+    pub verbose: bool,
+
+    #[clap(long, short)]
+    /// Print less output
+    pub quiet: bool,
+}
+
+#[async_trait]
+impl CliCommand<String> for Fmt {
+    fn command_name(&self) -> &'static str {
+        "Fmt"
+    }
+
+    async fn execute(mut self) -> CliTypedResult<String> {
+        self.command.execute().await
+    }
+}
+
+impl FmtCommand {
+    async fn execute(self) -> CliTypedResult<String> {
+        let exe = get_movefmt_path()?;
+        let mut cmd = Command::new(exe.as_path());
+        let input_opt = self.file_path;
+        let dir_opt = self.dir_path;
+        let config_path_opt = self.config_path;
+        let config_map = self.config;
+        let verbose_flag = self.verbose;
+        let quiet_flag = self.quiet;
+        let emit_mode = match self.emit_mode {
+            EmitMode::Overwrite => "overwrite",
+            EmitMode::NewFile => "new_file",
+            EmitMode::StdOut => "stdout",
+            EmitMode::Diff => "diff",
+        };
+        cmd.arg(format!("--emit={}", emit_mode));
+        if let Some(config_path) = config_path_opt {
+            cmd.arg(format!("--config-path={}", config_path.as_path().display()));
+        }
+        if verbose_flag {
+            cmd.arg("-v");
+        } else if quiet_flag {
+            cmd.arg("-q");
+        }
+        if !config_map.is_empty() {
+            let mut config_map_str = "".to_string();
+            let mut first = true;
+            for (key, value) in config_map {
+                if first {
+                    config_map_str = format!("{}={}", key, value);
+                    first = false;
+                } else {
+                    config_map_str = format!("{},{}={}", config_map_str, key, value);
+                }
+            }
+            cmd.arg(format!("--config={}", config_map_str));
+        }
+        if input_opt.is_some() {
+            cmd.arg(format!(
+                "--file-path={}",
+                input_opt.unwrap().as_path().display()
+            ));
+        } else if dir_opt.is_some() {
+            cmd.arg(format!(
+                "--dir-path={}",
+                dir_opt.unwrap().as_path().display()
+            ));
+        } else {
+            cmd.arg("--dir-path=./");
+        }
+        let to_cli_error = |e| CliError::IO(exe.display().to_string(), e);
+        let out = cmd.output().map_err(to_cli_error)?;
+        if out.status.success() {
+            let string_res = String::from_utf8(out.stdout);
+            if string_res.is_ok() {
+                eprint!("{}", string_res.clone()?.clone());
+                return Ok("ok".to_string());
```

Comment:
How come we don't return the output here? As in, take the CLI stdout and return that here instead of returning "ok".

Reply:
Currently, CLI will print out very string format to terminal such as
```
{
  "Result": "\u001b[2m2024-07-26T19:27:11.536061Z\u001b[0m \u001b[33m WARN\u001b[0m \u001b[2mmovefmt\u001b[0m\u001b[2m:\u001b[0m \n            Currently movefmt is still in the beta testing phase.\n            The formatting results of the beta version may be incompatible with the official release version.\n        \n2 files successfully formatted\n"
}
```
So I use `eprint` and return `ok` instead. Any suggestion how to make it better?

Reply:
Ahh I see, no worries then, those escapes are annoying.